music/src/music_rnn/model.ts

```diff
@@ -142,13 +95,43 @@ export class MusicRNN {
     return this.initialized;
   }
 
+  /**
+   * Instantiates data converter, attention length, chord encoder, and auxiliary
+   * inputs from the `MusicRNNSpec`.
+   */
+  private instantiateFromSpec() {
+    this.dataConverter = data.converterFromSpec(this.spec.dataConverter);
+    this.attentionLength = this.spec.attentionLength;
+    this.chordEncoder = this.spec.chordEncoder ?
+        chords.chordEncoderFromType(this.spec.chordEncoder) :
+        undefined;
+    this.auxInputs = this.spec.auxInputs ?
+        this.spec.auxInputs.map(s => aux_inputs.auxiliaryInputFromSpec(s)) :
+        undefined;
+  }
+
   /**
    * Loads variables from the checkpoint and instantiates the `Encoder` and
    * `Decoder`.
    */
   async initialize() {
     this.dispose();
 
+    if (!this.spec) {
+      await fetch(`${this.checkpointURL}/config.json`)
+          .then((response) => response.json())
```

Comment:
We should throw an error if the file isn't found.

Reply:
Will fix in subsequent PR.